queries/faust/highlights.scm

```diff
@@ -0,0 +1,239 @@
+; Identifiers
+(process) @variable.builtin
+
+(parameters
+  (identifier) @variable.parameter)
+
+(identifier) @variable
```

Comment:
flip these, you'll never have parameters highlighted properly (queries have a last-wins precedence)

Reply:
Ahh that explains the issue I was seeing with parameters - thanks!!